documentation/release_notes.rst

```diff
@@ -53,6 +53,8 @@ New in This Release
   with ``unseq`` or ``par_unseq`` policy when compiled by Intel® oneAPI DPC++/C++ Compiler
   with ``-fiopenmp``, ``-fiopenmp-simd``, ``-qopenmp``, ``-qopenmp-simd`` options on Linux.
   To avoid the issue, pass ``-fopenmp`` or ``-fopenmp-simd`` option instead.
+- Incorrect results may be produced with group reduce operation through ``std::reduce``
+  for 64-bit data types and `std::multiplies`` operation on GPUs with the Intel® C++ Compiler 2021.3
```

Comment:
Yes

Reply:
Not all: this error  happen when `__has_known_identity` used: when `if _ONEDPL_LIBSYCL_VERSION >= 50200`
Otherwise we don't use group operation for multiply.
And this mean all compiler versions from 2021.3

Reply:
Applied